ArduCopter/mode_acro_heli.cpp

```diff
@@ -45,14 +45,14 @@ void ModeAcro_Heli::run()
     switch (motors->get_spool_state()) {
     case AP_Motors::SpoolState::SHUT_DOWN:
         // Motors Stopped
-        attitude_control->reset_target_and_rate();
+        attitude_control->reset_target_and_rate(true);
```

Comment:
Shouldn't the argument read "false" since we don't want it to reset rates?

Reply:
yep, sorry